### PR TITLE
Avoid using namespace that is a reserved object

### PR DESCRIPTION
**Pull request by sdwerwed**
Related to this issue https://github.com/kubernetes/website/issues/24750

### CODE REVIEW

**Comment by sdwerwed** [comment]
I signed it

**Comment by sdwerwed** [comment]
I signed it

**Comment by sdwerwed** [comment]
I signed it

**Comment by sdwerwed** [comment]
I signed it

**Comment by sdwerwed** [comment]
I signed it

**Comment by sdwerwed** [comment]
I signed it

**Comment by sdwerwed** [comment]
I signed it

File: content/en/docs/tasks/run-application/run-replicated-stateful-application.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ on general patterns for running stateful applications in Kubernetes.
   [ConfigMaps](/docs/tasks/configure-pod-container/configure-pod-configmap/).
 * Some familiarity with MySQL helps, but this tutorial aims to present
   general patterns that should be useful for other systems.
+* Assumes you are using the default namespace or another namespace that does not contain any conflicting objects.
```
Comment (tengqm) [inline content/en/docs/tasks/run-application/run-replicated-stateful-application.md:42]:
```suggestion
* You are using the default namespace or another namespace that does not contain any conflicting objects.
```

Comment (sdwerwed) [reply content/en/docs/tasks/run-application/run-replicated-stateful-application.md]: Fixed and squashed

**Comment by sftim** [comment]
Thanks
/approve

**Pull request merged**